Mathlib/GroupTheory/SpecificGroups/Cyclic.lean

```diff
@@ -680,3 +680,80 @@ lemma not_isCyclic_iff_exponent_eq_prime [Group α] {p : ℕ} (hp : p.Prime)
   · exact False.elim <| h_cyc <| isCyclic_of_orderOf_eq_card g <| by aesop
 
 end Exponent
+
+section ZMod
+
+open Subgroup AddSubgroup
+
+variable {G H : Type*}
+
+/-- The isomorphism between `(Multiplicative G) / ⟨.ofAdd g⟩` and `Multiplicative (G / ⟨g⟩)`. -/
+def multiplicativeQuotientZpowersZmultiplesMulEquiv [AddCommGroup G] (g : G) :
+    Multiplicative G ⧸ zpowers (Multiplicative.ofAdd g) ≃* Multiplicative (G ⧸ zmultiples g) :=
+  MulEquiv.refl _
+
+lemma multiplicativeQuotientZpowersZmultiplesMulEquiv_mk_eq [AddCommGroup G] (g : G)
+    (h : Multiplicative G) :
+    multiplicativeQuotientZpowersZmultiplesMulEquiv g (QuotientGroup.mk h) =
+    .ofAdd (QuotientAddGroup.mk h) := rfl
+
+lemma multiplicativeQuotientZpowersZmultiplesMulEquiv_symm_mk_eq [AddCommGroup G] (g : G)
+    (h : G) :
+    (multiplicativeQuotientZpowersZmultiplesMulEquiv g).symm (QuotientAddGroup.mk h) =
+    QuotientGroup.mk (.ofAdd h) := rfl
```

Comment:
I'd be tempted to split these to a new PR: you're not using them, and the name possibly could do with some bikeshedding. It would also be nice to have the `Additive` versions.

Reply:
Yeah, they used to be used in the proof of `zmodCyclicMulEquiv` but not anymore.